lib/tooling_invoker/check_canary.rb

```diff
@@ -0,0 +1,29 @@
+module ToolingInvoker
+  class CheckCanary
+    include Mandate
+
+    def call
+      job = Jobs::TestRunnerJob.new(
+        "canary-#{SecureRandom.hex}",
+        'ruby',
+        'hello-world',
+        CANARY_SOURCE,
+        '1'
+      )
+      ProcessJob.(job)
```

Comment:
Is Ruby the fastest test runner? We could, for efficiency purposes, also consider other tracks. But Ruby is quite fast so that might not be necessary

Reply:
Yeah, I think it's pretty much the fastest.

Reply:
It's also the one we arguably have most control over. The consequence of this is that if this test runner breaks, the whole show will come tumbling down.

Reply:
Yeah. That's definitely a downside. Good thing we have CI. Looking at the ruby-test-runner repo, it doesn't have "golden tests": https://github.com/exercism/ruby-test-runner/ Might be useful to add them too, as they reduce the likelihood of something breaking.

Reply:
Yep. Nice idea.